### ISSUE TITLE
Can't connect to SSH Remote when update to 1.80.0

### ISSUE THREAD

**Issue by dacapoday** [microsoft/vscode-remote-release#8687]
### Output:
[10:07:54.314] Got request to download on client for {"platform":"linux","arch":"x64","destFolder":"/home/xxxxx/.vscode-server/bin/660393deaaa6d1996740ff4880f1bad43768c814"}
[10:07:54.314] Downloading VS Code server locally...
[10:07:54.336] > 
> 
[10:08:15.630] Resolver error: Error: XHR failed
    at y.onerror (vscode-file://vscode-app/c:/Users/xxxxx/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:77:1261)
[10:08:15.641] ------

------
Version: 1.80.0
Commit: 660393deaaa6d1996740ff4880f1bad43768c814
Date: 2023-07-04T15:06:02.407Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Code/1.80.0 Chrome/108.0.5359.215 Electron/22.3.14 Safari/537.36

**Comment by pertsevds** [comment]
I had a similar bug on MacOS.
My remote host is Linux.
After an update to 1.8.0 I tried to connect to it and it was timeout every time, while in terminal I was able to connect to it with ssh.
I've changed `remote.SSH.useLocalServer` to `false` and then i was able to connect to remote host.
Then after successful connection I changed `remote.SSH.useLocalServer` back to `true` and now it connects.

Version: 1.80.0
Commit: 660393deaaa6d1996740ff4880f1bad43768c814
Date: 2023-07-04T13:40:17.963Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0

**Comment by simonjenny** [comment]
I have a different Problem with SSH Remote Login after  Updating to 1.80.0. I do get the error: 

"Error: Not an authority that can be resolved. "

![grafik](https://github.com/microsoft/vscode-remote-release/assets/459369/0f8e0466-17a6-4162-890c-782baed6153b)

**Comment by alexshmmy** [comment]
Hello, I think I got the same problem. After updating to vscode 1.80.0 in my Mac, I cannot attach my Kubernetes pod to Visual studio code. It was working perfectly so far.

Error message:
![Screenshot 2023-07-07 at 22 16 54](https://github.com/microsoft/vscode-remote-release/assets/12173938/e722482a-d39b-45e0-a387-b5992b9bdf23)

**Comment by majames** [comment]
I was able to resolve the issue by downgrading to `1.79.x`. Binary can be downloaded from the release page here: https://code.visualstudio.com/updates/v1_79

**Comment by alexshmmy** [comment]
Thank you @majames, by downgrading to `1.79.x` solved the issue.

Don't also forget to disable automatic updates until the issue is fixed in `1.80` version, because it till automatically update to `1.80`
![Screenshot 2023-07-08 at 10 42 58](https://github.com/microsoft/vscode-remote-release/assets/12173938/add311a1-a2af-4e9f-b54c-6fc7a6dd09af)

**Comment by loviselu** [comment]
![image](https://github.com/microsoft/vscode-remote-release/assets/1033382/593982b6-de1f-479e-be58-f1a843eeae71)

I got the same problem. After updating to vscode 1.80.0.  Also can be resolved by downgrading to 1.79.x. 

Will it be fix in next version?

**Comment by chrmarti** [comment]
The "not an authority that can be resolved" error looks like https://github.com/microsoft/vscode/issues/187202. Please give the latest VS Code Insiders build a try where this should be fixed (https://code.visualstudio.com/insiders/) and open a new issue if it persists. Thanks.

**Comment by folger-fan** [comment]
> The "not an authority that can be resolved" error looks like [microsoft/vscode#187202](https://github.com/microsoft/vscode/issues/187202). Please give the latest VS Code Insiders build a try where this should be fixed (https://code.visualstudio.com/insiders/) and open a new issue if it persists. Thanks.

I have tried the latest vscode-insiders。Also has this problem。
![image](https://github.com/microsoft/vscode-remote-release/assets/7549667/2d0fa983-a162-4961-8078-1bf5f16d7234)

**Comment by loviselu** [comment]
> The "not an authority that can be resolved" error looks like [microsoft/vscode#187202](https://github.com/microsoft/vscode/issues/187202). Please give the latest VS Code Insiders build a try where this should be fixed (https://code.visualstudio.com/insiders/) and open a new issue if it persists. Thanks.

Thank for your reply.  I got The same error after use the latest VS Code Insiders build.  This is an example remote url of my case :
```
vscode://vscode-remote/ssh-remote+root@101.135.113.22:22/workspace
```

**Comment by minostauros** [comment]
> ![image](https://user-images.githubusercontent.com/1033382/252571826-593982b6-de1f-479e-be58-f1a843eeae71.png)
> 
> I got the same problem. After updating to vscode 1.80.0. Also can be resolved by downgrading to 1.79.x.
> 
> Will it be fix in next version?

Current workaround for this specific case is to remove `@` from context names. ( https://github.com/microsoft/vscode-remote-release/issues/8710#issuecomment-1636149886 )
